test-fixtures/generate-metrics.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 BUILD_LOG_FILE="$1"
 TYPE_LOG_FILE="$2"
-THREESHOLD_UNIT_TEST=51
-THREESHOLD_XCUITEST=50
+THREESHOLD_UNIT_TEST=29
+THREESHOLD_XCUITEST=28
```

Comment:
I see your latest build failed because of the threshold. I try to give a bit of wiggle room for warnings (+2-3). Depending on the workflow, warnings varies as we can see in firefox-ios-alerts channel.